src/main/java/core/basesyntax/WorkWithEnterList.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class WorkWithEnterList {
+    public WorkWithEnterList(OperationsStrategy operationsStrategy) {
+        this.operationsStrategy = operationsStrategy;
+    }
+
+    private OperationsStrategy operationsStrategy;
+
+
+    public String workWithEnterList(List<String> entery) {

```

Comment:
The method name `workWithEnterList` could be more descriptive. Consider renaming it to something like `processEntries` to better convey its purpose.